wolfcrypt/src/asn.c

```diff
@@ -9697,7 +9703,9 @@ static int DecodeBasicOcspResponse(byte* source, word32* ioIndex,
             return ASN_PARSE_E;
 
         InitDecodedCert(&cert, resp->cert, resp->certSz, heap);
-        ret = ParseCertRelative(&cert, CERT_TYPE, VERIFY, cm);
+        /* Don't verify if we don't have access to Cert Manager. */
+        ret = ParseCertRelative(&cert, CERT_TYPE,
```

Comment:
The function in question here is DecodeBasicOcspResponse which is not public.

The code currently turns off verification if cm is NULL.  I'd rather have an explicit parameter to turn off verification to prevent programming mistakes that lead to accidental no verifies.